purchase_console/static/js/console_frontend.js

```diff
@@ -1,23 +1,44 @@
 (function() {
     "use strict";
-        console.log('All Loaded Ok!');
-        $(document).on('click', '.js_action_management .js_action_btn', function () {
-            var $data = $(this).parents(".js_publish_management");
-            console.log($data);
-            console.log($data.data('id'));
+    console.log('All Loaded Ok!');
```

Comment:
Was added in order to debug the loading of the js scripts.
I will remove throughout the development.